playwright/tests/collapse-slot.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { expect, test } from '@playwright/test';
+import { articleWithCollapsedSlots } from '../fixtures/pages/articles';
+import { cmpAcceptAll } from '../lib/cmp';
+import { assertHeader, waitForGAMResponseForSlot } from '../lib/gam';
+import { loadPage } from '../lib/load-page';
+
+const { path } = articleWithCollapsedSlots;
+
+test.describe('Ad slot removal', () => {
+	test(`Empty ad slots should be removed from the DOM`, async ({ page }) => {
```

Comment:
Maybe a small thing, but should the test now not be described as `Empty ad slots should be NOT removed from the DOM, but be invisible`